lib/libspl/include/rpc/xdr.h

```diff
@@ -60,6 +60,8 @@ typedef struct xdr_bytesrec {
 #define	XDR_RDMAGET	4
 #define	XDR_RDMASET	5
 
+#ifndef WITH_TIRPC
```

Comment:
This #ifdef can go since we will need to reuse our xdr_control as I suggested below.